cljfmt/src/cljfmt/core.cljc

```diff
@@ -369,15 +382,15 @@
            current-namespace (some-> as-zloc z/leftmost z/sexpr)
            grandparent-node  (some-> as-zloc z/up z/up)
            parent-namespace  (when-not (ns-require-form? grandparent-node)
-                               (first (z/child-sexprs grandparent-node)))]
+                               (first (child-sexprs grandparent-node)))]
```

Comment:
This line doesn't need to be changed, as it's Clojure only.

Reply:
OTOH, with this, all JVM clj code will consistently use the same symbols. Using `child-sexprs` in some places but `z/child-sexprs` in others can be confusing, leading one to think there's a reason for that.

Same for `edn`

Can change, just confirm

Reply:
That's not a bad idea, but it's a job for another PR. We want the diff to show changes only relevant to the current PR.